04_environment/index.tex

```diff
@@ -1 +1,400 @@
+
+The focus of deer hunting is higher payoffs therefore it has better returns but all of its tier thresholds are significantly higher and it also has a “zero” tier. Therefore it is possible for a single island to go deer hunting without enough resources to reach the hunting location leading to no return.
+
+\begin{equation}
+Increments\ of\ catching\ n\ deer's=
+\left( \begin{array}{ll@{}}
+n=0 -> \Delta^0 = 1 \\
+n=1 -> \Delta^1 = 0.8^1 \\
+n=2 -> \Delta^2 = 0.8^2 = 0.64 \\
+n=2 -> \Delta^3 = 0.8^3 = 0.512 \\
+n=4 -> \Delta^4 = 0.8^4 = 0.4096\\
+\end{array} \right) 
+\end{equation}
+
+\begin{equation}
+Cumulative\ deer \ cost \sum_{n=0}^{n} \Delta^{n} = \Delta^{0} + \Delta^{1} + \Delta^{2} + \Delta^{3} + \Delta^{4} + ....
+\end{equation}
+
+\begin{equation}
+Cumulative\ fishing\ cost \ = \ \sum_{n=1}^{n} \Delta^{n} = \Delta^{1} + \Delta^{2} + \Delta^{3} + \Delta^{4} + \Delta^{5} + ....
+\end{equation}
+
+Equations that show the formulation of the decay function for tier system.\\
+
+The tier system is shown above, where n stands for the tiers, with decay costs ($\Delta$) of 0.8 for both the deer hunt and fishing. Fishing cost starts at n=1 due to the reduced cost.\\
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=0.9\textwidth]{04_environment/Images/Foraging Tier System.PNG}
+    \caption{Foraging Tier System}
+    \label{Images:Foraging Tier System}
+\end{figure}
+
+As the tiers increase the cost per animal decreases but the total cost increases up to a daily catch limit. Thus if teams collaborate, they could invest less as individuals to get a good return through deer hunting, however if they invest too much then they will over spend as they reach the daily max catch. Thus it would be beneficial for some teams to go fishing which has a higher maximum catch per day but comes with lower returns (thus there is a need for some self sacrifice to be done to reach maximum welfare). \\ 
+
+Fishing has similar functions with the key difference being the Tiers start at 1 and Normal distribution. \\
+
+\newpage
+\subsection{Examples Distributions}
+\subsubsection{Example of the distribution for foraging return (Deer hunting - payoff dominant)}
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/Images/Distribution of Foraging returns Deer Hunting.PNG}
+    \caption{Deer hunting payoff dominant Distribution for foraging return}
+    \label{Images:Distribution of Foraging returns Deer Hunting}
+\end{figure}
+
+To increase the risk we have used a Bernoulli Random Variable (D) that makes it possible for the hunt to not return any resources. Whilst the Exponential Decay (W) is to disincentive too much resources being placed in hunting and hoping for a high return. However this all comes with the benefit of significantly higher (2x) returns than fishing.\\ 
```

Comment:
go to math mode for variable/constants -- as suggested by the guidelines

Reply:
same with unnecessary \\ at the end - does not belong at the end of every paragraph